.github/workflows/push.yml

```
@@ -39,7 +39,7 @@ jobs:
           java-version: "17.0"
       - uses: subosito/flutter-action@v2
         with:
-          flutter-version: "3.29.3"
+          flutter-version: "3.32.8"
           channel: "stable" # or: 'beta', 'dev' or 'master'
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)